steps/step-copy-file/0.0.3/step.yml

```diff
@@ -0,0 +1,32 @@
+title: copy-file
+summary: |
+  Step to copy a file from one folder to another. You can use this for copying configuration files fromm archived location to a specific directory in the workspace.
+description: |
+  Step to copy a file from one folder to another. You can use this for copying configuration files from archived location to a specific directory in the workspace.
+website: https://github.com/satyasrikantmantha/bitrise-step-copy-file
+source_code_url: https://github.com/satyasrikantmantha/bitrise-step-copy-file
+support_url: https://github.com/satyasrikantmantha/bitrise-step-copy-file/issues
+published_at: 2017-08-30T15:46:24.272686989+05:30
+source:
+  git: https://github.com/satyasrikantmantha/bitrise-step-copy-file.git
+  commit: 107bcae6277f8809ede735695fd8e0334d6b2254
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    is_required: true
+    title: Source file path
+  source_file_path: Specify source file path
```

Comment:
`source_file_path: Specify source file path` this means you provide default value (`Specify source file path`) for `source_file_path` input.
I think this should be the input's `description`, read more about step input envs: https://github.com/bitrise-io/bitrise/blob/master/_docs/bitrise-yml-format-spec.md#environment-properties